MediaManager.Abstractions/IVideoPlayer.cs

```diff
@@ -21,5 +23,18 @@ public interface IVideoPlayer : IPlaybackManager
         /// The aspect mode of the video
         /// </summary>
         VideoAspectMode AspectMode { get; set; }
+        
+        int TrackCount { get; }
```

Comment:
Can't you get this through the Queue?

Reply:
TrackCount is to store the number of available tracks in a video:
Android: https://developer.xamarin.com/api/member/Android.Media.MediaPlayer.GetTrackInfo/
iOS: https://developer.xamarin.com/api/property/MonoMac.AVFoundation.AVMediaCharacteristic.Audible/